README.md

```diff
@@ -22,57 +30,92 @@
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-# 📖 [your_project_name] <a name="about-project">Hello Microverse</a>
 
-**[your_project__name]** This is a basic HTML & CSS project to practice all the tools that a wiil need to use in the future.
+# 📖 [Hello Microverse project] <a name="about-project"></a>
+
```

Comment:
- [ ] Your repo doesn't have the `package.json` and  `package.lock.json` files, this insinuates that you didn't run the necessary commands in you terminal, such as the `npm init -y` and the others on that page, this will ensure a proper linters and dependency configuration.